### ISSUE TITLE
Add Aria-Label and Alt Text to Promoblock

### ISSUE THREAD

**Issue by dlescarbeau** [NCIOCPL/ncids#820]
### Issue description
- The Promoblock should pass Axe Devtools accessibility scans 

### What's the expected result?
- Promoblock should pass accessibility scans

### What's the actual result?
- Promoblock has issues with accessibility related to link aria-labels and image alt text being missing
